.github/workflows/mobilecoin-dispatch-dev-test.yaml

```
@@ -7,11 +11,45 @@ on:
         description: "Target Namespace"
         type: string
         required: true
+      ingest_color:
+        description: "Fog Ingest blue/green"
+        type: string
+        required: true
+        default: blue
+      fog_distribution:
+        description: "Run fog-distribution test"
+        type: boolean
+        required: false
+        default: false
+      testing_block_v0:
+        description: "Run block v0 tests"
+        type: boolean
+        required: false
+        default: true
+      testing_block_v2:
+        description: "Run block v2 tests"
+        type: boolean
+        required: false
+        default: true
+      client_auth_enabled:
+        description: "Is client auth enabled?"
+        type: boolean
+        required: false
+        default: false
 
 jobs:
-  list-values:
-    runs-on: [self-hosted, small, Linux]
-    steps:
-    - name: values
-      run: |
-        echo namespace ${{ github.event.inputs.namespace }}
+  test:
+    uses: ./.github/workflows/mobilecoin-workflow-dev-test.yaml
+    with:
+      namespace: ${{ github.event.inputs.namespace }}
+      ingest_color: ${{ github.event.inputs.ingest_color }}
+      fog_distribution: ${{ github.event.inputs.fog_distribution == 'true' && 'true' || 'false' }}
+      testing_block_v0 : ${{ github.event.inputs.testing_block_v0 == 'true' && 'true' || 'false' }}
+      testing_block_v2: ${{ github.event.inputs.testing_block_v2 == 'true' && 'true' || 'false' }}
```

Optional/Equivalent: Pass the stringified boolean directly
```suggestion
      fog_distribution: ${{ github.event.inputs.fog_distribution }}
      testing_block_v0 : ${{ github.event.inputs.testing_block_v0 }}
      testing_block_v2: ${{ github.event.inputs.testing_block_v2 }}
```

---

Yes and no.  I had some issues when trying to just pass the booleans here.  I'd like to go with this pattern for now since I know it works and come back and see if I can clean this up.